mzLib/FlashLFQ/FlashLfqEngine.cs

```diff
@@ -471,6 +474,74 @@ private void QuantifyMs2IdentifiedPeptides(SpectraFileInfo fileInfo)
             _results.Peaks[fileInfo].AddRange(chromatographicPeaks.ToList());
         }
 
+        /// <summary>
+        /// Used by the match-between-runs algorithm to determine systematic retention time drifts between
+        /// chromatographic runs.
+        /// </summary>
+        private RetentionTimeCalibDataPoint[] GetRtCalSpline(SpectraFileInfo donor, SpectraFileInfo acceptor, MbrScorer scorer)
+        { 
+            Dictionary<string, ChromatographicPeak> donorFileBestMsmsPeaks = new();
```

Comment:
How often is this method called? Does it make sense to store these anchor peptides instead of calculating them on the fly every time? This is a minor point and not terribly pertinent to your PR.

Reply:
It's called once for every donor/acceptor file pair. The results are held for as long as they're needed. This method is never called twice with the same donor/acceptor file pair.